### ISSUE TITLE
Multiple SauceLabs Videos

### ISSUE THREAD

**Issue by winstondz** [reportportal/reportportal#1008]
Is there a way we can embed multiple saucelabs videos in a single test using more than one SLID values? For example, I need to run a larger test case, as part of which I launch a new browser for different flows in a single end to end test case. So it would be good to get all videos in the same test? Is there a way we can do this?

**Comment by evjlobanova** [comment]
We have such a feature in our backlog, but it is difficult to say ETA for the feature.

Anyway, do you use SauceLabs integration in ReprtPortal?

**Comment by winstondz** [comment]
Yes, We do. For now I create a weblink and post it. This works because it navigates back to saucelabs where I can see the corresponding video. I was thinking if I can avoid going back to saucelabs and just see the videos in report portal itself

**Comment by evjlobanova** [comment]
Thank you for your response.
